modules/localplay/upnp/upnpplayer.class.php

```diff
@@ -69,8 +69,8 @@ private function Playlist()
      */
     public function __construct($name = "noname", $description_url = "http://localhost")
     {
-        require_once AmpConfig::get('prefix') . '/modules/localplay/upnp/upnpdevice.php';
-        require_once AmpConfig::get('prefix') . '/modules/localplay/upnp/upnpplaylist.php';
+        require_once AmpConfig::get('prefix') . '/modules/localplay/upnp/UPnPDevice.php';
+        require_once AmpConfig::get('prefix') . '/modules/localplay/upnp/UPnPPlaylist.php';
```

Comment:
I had a dilemma here. PSR-0 requires the filename to match the one of the defined class name. So we should:
* Either deviate from PSR-0 here
* Rename `UPnPDevice` class to `upnpdevice`, which I don't like much personnally
* Rename the file names

Not sure which is the best.